resume-buddy/src/main/webapp/careers.json

```diff
@@ -0,0 +1,443 @@
+[
```

Comment:
This is great! Where did you get the careers and universities .json files? We may want to give the sources credit somewhere.

Reply:
I found lists here: https://gist.github.com/hakimelek/147f364c449104ba66da9a3baca9d0c0 andhttps://gist.github.com/mbejda/19012b99a12e9d014389.
How should I give credits to sources?